zyte_common_items/processors.py

```diff
@@ -345,3 +345,21 @@ def metadata_processor(metadata: BaseMetadata, page):
     if page.metadata_cls is None:
         return None
     return metadata.cast(page.metadata_cls)
+
+
+def string_processor(value: str) -> Union[str, None]:
+    """Processor for string values"""
+    if isinstance(value, str):
+        return value.strip()
+    return value
```

Comment:
Perhaps we should return `None` if value is not a string instance...?

Reply:
I feel like we should convert it to `str` unless it is `None`.

I also wonder if we should return `None` if `not value.strip()`.